### ISSUE TITLE
Fix player invisible after teleportation ?

### ISSUE THREAD

**Issue by sathonay** [Electroid/SportPaper#70]
Hello, i don't know if you fixed this yet.

Subject: Player invisible for other players after teleportation ([youtube](https://www.youtube.com/watch?v=RL6pmNHjPfo) or [MC-97010](https://bugs.mojang.com/browse/MC-97010)) is an exemple).
But i was thinking about when a player (far from you) gets teleported to you, and for a longer or shorter period of time you can't see him.

I found a fix for that:
- [first](https://github.com/sathonay/nPaper/commit/df76206e7af9972688a49019858f12385341278d#diff-9e1c701a791043757bec054e6b2304af26acfd7f54d778ea16313e016071f43a) | [second](https://github.com/sathonay/nPaper/commit/50dd6914e50a2299b54e79d5d81315b94f11476b#diff-9e1c701a791043757bec054e6b2304af26acfd7f54d778ea16313e016071f43a)

It's basiclly:
```patch
@@ -147,7 +147,7 @@ public void track(List list) {
                        this.v = 0;
                        // CraftBukkit start - Refresh list of who can see a player before sending teleport packet
                        if (this.tracker instanceof EntityPlayer) {
-                           this.scanPlayers(new java.util.ArrayList(this.trackedPlayers));
+                           this.scanPlayers(new java.util.ArrayList(this.tracker.world.players); // scan all players of the world (that's fix invisibility after teleportation)
                        }
                        // CraftBukkit end
                        packet = new PacketPlayOutEntityTeleport(this.tracker.getId(), i, j, k, (byte) l, (byte) i1, tracker.onGround, tracker instanceof EntityFallingBlock || tracker instanceof EntityTNTPrimed); // Spigot - protocol patch
```

Tell me if you found this useful or not, and if it's working for you.
(Sorry if my english isn't that good ':) i'm french)

**Comment by linsaftw** [comment]
I will implement the patch in a PR.

**Comment by linsaftw** [comment]
Fixed in #76.

JAR with the fixes -> https://github.com/linsaftw/SportPaper/releases/tag/3112d88

**Comment by Electroid** [comment]
Closed by #76
